freqtrade/freqtradebot.py

```diff
@@ -148,6 +148,8 @@ def cleanup(self) -> None:
 
         self.check_for_open_trades()
 
+        self.strategy.ft_bot_cleanup()
```

Comment:
should probably also be called from backtesting ...

Reply:
Where would that be called? I can't seem to find where things are cleaned up in the `Backtesting` object.

Reply:
Just at the bottom of `Backtesting.start`?

Reply:
> should probably also be called from backtesting ...

Can you be more specific? FreqAI backtesting is all single threaded. It does not require this kind of shut down sequence.

Reply:
well my assumption was that `ft_bot_cleanup()` may be used to clean up other things too over time.

if we discover something else that'll have to be shutdown (with or without freqAI context) - will you remember to check if this is actually called everywhere? I know i probably won't ... 

Leave it for now - @robcaulk  is right, we'll not need this in backtesting for now.